src/components/OSCALControl.js

```diff
@@ -85,14 +85,14 @@ export default function OSCALControl(props) {
               control={props.control}
               controlId={props.control.id}
               parameters={props.control.params}
-              implReqStatements={props.implReqStatements}
+              implementedRequirement={props.implementedRequirement}
               componentId={props.componentId}
               modificationAlters={props.modificationAlters}
               modificationSetParameters={props.modificationSetParameters}
               key={part.id ?? `part-${index}`}
-              implementedRequirement={props.implementedRequirement}
               isEditable={props.isEditable}
-              onFieldSave={props.onFieldSave}
```

Comment:
Since we're not using the simple field a more generic `onRestSuccess` was used instead.